.travis.yml

```
@@ -30,17 +30,17 @@ os:
   - windows
 julia:
   - 1.0  # LTS
-  - 1.4  # Latest release
+  - 1.3  # latest release with old way of downloading artifacts
+  - 1.4  # Latest stable release
+  - 1.5  # Latest release
```

Let's just make this dynamic. I don't think we need to test on both 1.4 and 1.5 as there isn't any special logic here that differs between 1.4 and 1.5
```suggestion
  - 1    # Latest release
```

---

Well, Julia 1.5 uses Package server by default, and Julia 1.4 not, so this allows us to see potential problems with Package Server related to the Artifact, but also in general.
In Julia 1.4 Package server is not used by default.
Testing on both of them already helped me discover some weird behavior of Package Server.
But we can leave it only on `1`.